### DISCUSSION TITLE
Multi-select filter shows full taxonomy

### DISCUSSION THREAD

**Discussion by boing0** [microsoft-search/pnp-modern-search#3430] [Q&A] [answered]
I have RefinableString04 mapped to a People column in my filter that behaves normally when it's a single-select, but when I change it to multi-select, it's showing the full taxonomy. If I change it to AutoRefinableString04 nothing shows up in the filter part. What am I doing wrong? I'm using version 4.10.1.

Single-select:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/8752412/711817e7-7d1c-4a4d-b417-9ff4dab1abfb)

Filter shows the names:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/8752412/05330fee-de04-4257-abf6-fc31f7478cd7)

Change it to multi-select:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/8752412/b67d60fa-4336-4fc9-a818-646604838d33)

And it shows the full taxonomy:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/8752412/a437a0ac-6c36-4328-b1c2-87b56506094c)

**Comment by kasperbolarsen** [comment]
Can you share the mapping of the RefinableString04?

**Reply by boing0** [reply]
![image](https://github.com/microsoft-search/pnp-modern-search/assets/8752412/0aa077db-c085-4d58-8933-7061e7dc1806)

**Comment by kasperbolarsen** [comment, marked as answer]
Thanks :-)
Could you map RefinableString06 to OWS_Leadbanker and see if it will solve the issue? 

I am using that mapping here, and the names are shown correct:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/e8137be9-77f4-496b-9765-17989444a177)

**Reply by boing0** [reply]
Thanks Kasper! I'll need to make the request (I don't have admin access in that tenant).

One other question about People -- is it possible to merge/collapse results based on their email address? We changed display names at some point from "Firstname Lastname" to "Firstname Lastname (Department Name)". That means we have the same users appearing twice even though the email address and ID are the same. That looks like this in the Filter (and is why I need to allow multi-select on that field, to get all results for the same user):

![image](https://github.com/microsoft-search/pnp-modern-search/assets/8752412/21c1db46-86ea-42a6-9e11-d7b42f09a8b3)

**Reply by boing0** [reply]
Thanks Kasper, that was the correct field to map, also took care of my duplicates.
